src/app/public/modules/numeric/numeric.options.ts

```diff
@@ -3,6 +3,6 @@ export class NumericOptions {
   public format = 'number';
   public iso = 'USD';
   public minDigits?: number;
-  public truncate? = true;
-  public truncateAfter? = 0;
+  public truncate?: boolean = true;
```

Comment:
I thought at one point you said we didn't need the type declarations if the property was initialized. Why are you adding them here?

Reply:
You are correct, however, the later version of TypeScript screams at me if we don't provide it (and use the "optional" marker `?`).

Reply:
You are correct, however, the later version of TypeScript screams at me if we don't provide the type (and we also wish to use the "optional" marker `?`).

Reply:
Ok. Sounds good.